src/redux/actions/user.js

```diff
@@ -63,11 +71,11 @@ export function doAuthenticate(appVersion, os = null, firebaseToken = null) {
     });
 
     Lbryio.authenticate()
-      .then(user => {
+      .then(accessToken => {
         // analytics.setUser(user);
         dispatch({
           type: ACTIONS.AUTHENTICATION_SUCCESS,
-          data: { user },
+          data: { accessToken },
```

Comment:
With this, do we have to explicitly call `/user/me` after authentication in order to retrieve the user? This seems like an extra step that I'd like to avoid. Why can't we obtain both the user and the accessToken?

Reply:
There was a bug where "user" in the reducer was being populated with an accessToken and then overwritten with the "user" object. Here I am, I think, just making it work as the state intends.
We can run the question of whether authenticate should also return the user by @tiger5226 @kauffj @seanyesmunt etc.

Reply:
I am not sure I understand. `/user/me` does return the user. What API are you using to authenticate?

Reply:
@jessopb why not populate the `user` object _and_ the `accessToken` here?